dashboard/config/routes.rb

```diff
@@ -724,8 +724,20 @@
 
     resources :zendesk_session, only: [:index]
 
-    post '/report_abuse', to: 'report_abuse#report_abuse'
-    get '/report_abuse', to: 'report_abuse#report_abuse_form'
+    # Public abuse report form
+    controller :report_abuse do
+      get  '/report_abuse', action: :report_abuse_form
+      post '/report_abuse', action: :report_abuse
+    end
```

Comment:
I ended up using the same patter as in here:
https://github.com/code-dot-org/code-dot-org/pull/65099/files#diff-3fffe83c5f87f14c4c59c00fc3556305f7d4a28ee7748b688700228b3c64e70dL649-L653

Reply:
I dediced to follow this existing pattern.